karaf/init/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/osgi/ToscaTypePlanTransformerClassloaderWrapper.java

```diff
@@ -17,7 +20,7 @@
  * classloader for this bundle, rather than whatever may happen to be in place at the
  * point of call.
  */
-public class ToscaTypePlanTransformerClassloaderWrapper implements BrooklynTypePlanTransformer {
+public class ToscaTypePlanTransformerClassloaderWrapper extends ToscaTypePlanTransformer implements BrooklynTypePlanTransformer {
```

Comment:
I did have to add a couple of extra inherited methods, the `createApplicationSpecFromTopologyId` (which is used by the `ToscaEntitySpecResolver`) and `createSpec`.  I'll add that note about the inheritance, you're right, it's worth highlighting for future readers.